src/api/app/controllers/webui/packages/assignments_controller.rb

```diff
@@ -14,6 +14,14 @@ def create
         redirect_to package_show_path(@package.project, @package)
       end
 
+      def destroy
+        assignment = authorize Assignment.find(params['id'])
```

Comment:
Adding the `authorize` to this line should belong to the last commit. We could also merge the last commit "Authorize who can unassign people"  into the first one "Implement the unassignment for a package".

Also, introducing the `destroy` method makes not necessary for all actions of this controller to call the `set_assignee` before filter. On line 6:

```
before_action :set_assignee, except: :destroy
```

Reply:
true

Reply:
fixed

Reply:
I merged the authorization change into the implementation